package.json

```diff
@@ -47,7 +47,7 @@
     "@storybook/addon-links": "^6.4.7",
     "@storybook/react": "^6.4.7",
     "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
+    "@testing-library/react": "^11.2.7",
```

Comment:
Did you not need to update `devDependencies` of React to something later?

Reply:
no, this version has `react: *` as peerDep